assets/styles/user-profile.scss

```diff
@@ -0,0 +1,518 @@
+        //     //width: auto;
+
+        //     .list-item__preview-header {
+        //         background-color: unset;
+        //         color: unset;
+        //     }
+        // }
+        &--new {
+            width: 100%;
+        }
+    }
+
+    > .row {
+        display: flex;
+        align-content: flex-start;
+        overflow: auto;
+    }
+}
+
+.profile-content__footer {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    background-color: var(--color-background-content-item);
+    padding: var(--space--1-5) var(--space--2);
+    border-block-start: 1px solid var(--color-line--light);
+    gap: var(--space--1);
+    grid-row: 2 / 3;
+    @include md {
+        padding: var(--space--1-5) var(--space--3);
+    }
+}
+
+.profile__mobile-close {
+    position: fixed;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    inset-block-start: var(--space--1);
+    inset-inline-end: var(--space--1);
+    width: 32px;
+    height: 32px;
+    .icon-button {
+        width: 32px;
+        height: 32px;
+    }
+}
+.profile__profile-content {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    overflow-y: hidden;
+    position: relative;
+    background-color: var(--color-background--muted);
+}
+.profile__profile-content-header {
+    position: relative;
+    display: flex;
+    align-items: center;
+    background: var(--color-background-content-item);
+    box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.15);
+    padding: 0px;
+    padding-inline-start: var(--space--2);
+    height: var(--profile-header-height);
+    z-index: 1;
+    flex-shrink: 0;
+    flex-grow: 0;
+    @include md {
+        padding-inline-start: var(--space--3);
+    }
+}
+.profile__profile-content-close {
+    width: var(--profile-header-height);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    align-self: stretch;
+    background-color: var(--color-background-content-item);
+    cursor: pointer;
+    border: none;
+    border-inline-start: 1px solid var(--color-line--x-light);
+    margin-inline-start: auto;
+}
+
+.profile__profile-content-title {
+    margin: 0;
+    line-height: 1.1;
+    font-size: var(--profile-header-title-font-size);
+}
+
+.profile__profile-content-main {
+    display: flex;
+    overflow-y: auto;
+    flex-grow: 1;
+}
+.profile__profile-content-footer {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    padding: var(--space--1-5) var(--space--2);
+    border-block-start: 1px solid var(--color-line--light);
+    gap: var(--space--1);
+}
+
+@media (max-width: 768px) {
+    .profile__profile-content-header {
+        --profile-header-height: var(--navbar-height--mobile);
+    }
+    .profile__profile-content-title {
+        --profile-header-title-font-size: var(--text-size--large);
+    }
+    .profile__profile-content-close {
+        display: none !important;
+    }
+}
+
+// Backup
```

Comment:
If there's a reason to keep commented code around, that's okay, otherwise.... more commented code!